### PR TITLE
feat: add grafana dashboard samples using GMP metrics

### PR DESCRIPTION
**Pull request by leonzz**
#### Description
- Add a folder containing sample Grafana dashboards using GMP metrics data, to be used together with Anthos clusters on VMware and on bare metal public documentation.

#### Change summary
The following dashboard json files are added to monitor their respective resource types:
- cluster-capacity.json
- control-plane.json
- daemonset.json
- deployment.json
- etcd.json
- kubelet.json
- node.json
- pod.json
- statefulset.json

### CODE REVIEW

**Comment by leonzz** [comment]
cc @waterloong @catherinef-dev

**Review by waterloong — APPROVED** [review]

**Review by waterloong — APPROVED** [review]

**Review by NimJay — APPROVED** [review]
Thanks for adding an associated screenshot for each JSON configuration. 😄 
Thanks for also including a [README.md](https://github.com/GoogleCloudPlatform/anthos-samples/blob/188ebe4939ba30d468580212d1b90eb13f3261aa/gmp-grafana-dashboards/README.md) with a link for:
1. [Managed Service for Prometheus on bare metal](https://cloud.google.com/anthos/clusters/docs/bare-metal/latest/how-to/log-monitoring#using-managed-service-for-prometheus)
2. [Managed Service for Prometheus on VMware](https://cloud.google.com/anthos/clusters/docs/on-prem/latest/how-to/logging-and-monitoring#using-managed-service-for-prometheus)

**Questions** for @leonzz:
**1.** Once this PR is merged, do you plan on referencing these JSON configurations inside a cloud.google.com docs page? If so, I highly recommend (later) updating the README.md to include those cloud.google.com pages.
**2.** What about the screenshots? (We should instead host cloud.google.com images inside cloud.google.com itself.)

Thinking out loud (no response needed): it would be nice if we had some way of testing (unit/integration) these configurations, but that sounds like a large and complex effort.

Approved. I trust that you've done the appropriate testing (judging from your screenshots).
I'll let you handle merging.

File: gmp-grafana-dashboards/control-plane.json [hunk 1]
```diff
@@ -0,0 +1,1074 @@
...
+      "2d",
+      "7d",
+      "30d"
+    ]
+  },
+  "timezone": "browser",
+  "title": "Control Plane Status",
+  "uid": "de2dd5401f5675f32b9504cb066276b8",
+  "version": 1
+}
```
Comment (NimJay) [inline gmp-grafana-dashboards/control-plane.json:1074]: **Question:** Are these configurations specific to Anthos?
Or are they usable in any Kubernetes-Prometheus setup?

Comment (leonzz) [reply gmp-grafana-dashboards/control-plane.json:1074]: most of the configurations would work for any kubernetes prometheus setup. however there're a few places, i.e. in `pod.json` and `cluster-capacity.json`, where we used some metrics that's specific to Anthos clusters, with the [PromQL for Cloud Monitoring metrics](https://cloud.google.com/stackdriver/docs/managed-prometheus/promql) name mapping convention.

**Comment by leonzz** [comment]
Thanks  @NimJay  for the feedback and the trust! Re on your questions:

> 1. Once this PR is merged, do you plan on referencing these JSON configurations inside a cloud.google.com docs page? 

Yes, that's exactly the plan. I've already included the intended doc page in `README.md` on line 6. Once we submit this PR, we'll make the change to update doc as well.

> 2. What about the screenshots? 

Since these are samples and we expect customers to make changes based on their needs, we won't reference all the screenshots in cloud.google.com doc page. So i think it still makes sense to have the screenshots here so that customers have a preview of what's in the samples. For the screenshots that got referenced in the doc, we will also host a copy in cloud.google.com.

**Pull request merged**